src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -532,7 +545,7 @@ private async Task CreateInternal(BlittableJsonReaderObject bjro, SubscriptionCr
                 var node = Database.SubscriptionStorage.GetResponsibleNode(serverContext, name);
                 if (node != null && node != ServerStore.NodeTag)
                 {
-                    await WaitForExecutionOnSpecificNode(serverContext, ServerStore.GetClusterTopology(serverContext), node, subscriptionId);
+                    await WaitForExecutionOnSpecificNode(serverContext, ServerStore.GetClusterTopology(serverContext), node, index);
```

Comment:
also, what are you waiting for here?

Reply:
I guess we want to wait for the index commit of `PutSubscriptionCommand` on it's relevant node, before returning result, because we might get `SubscriptionDoesNotExistException` exception otherwise